sdcm/cluster_k8s/__init__.py

```diff
@@ -431,15 +450,18 @@ def deploy_scylla_manager(self, pool_name: str = None) -> None:
             if scylla_operator_image_tag:
                 values.set('controllerImage.tag', scylla_operator_image_tag)
 
-            # Install and wait for initialization of the Scylla Manager chart
-            LOGGER.info("Deploy scylla-manager")
             self.kubectl(f'create namespace {SCYLLA_MANAGER_NAMESPACE}')
 
             # TODO: usage of 'cordon' feature below is a workaround for the scylla-operator issue #496
             # where it is not possible to provide node/pod affinity for the scylla server
             # which gets installed for the scylla-manager deployed by scylla-operator.
             to_cordon = ", ".join(['loader-pool', 'monitoring-pool', 'scylla-pool'])
             with CordonNodes(self.kubectl, f"{self.POOL_LABEL_NAME} in ({to_cordon})"):
+                # NOTE: sleep for some time to make kube scheduler fit the cordon nodes time range
```

Comment:
It will be proper to fix CordonNodes to wait till node gets cordoned and uncordonned

Reply:
Done